packages/client/hmi-client/src/types/Types.ts

```diff
@@ -792,6 +792,7 @@ export interface TerariumAssetThatSupportsAdditionalProperties extends TerariumA
 export interface ModelDistribution {
     type: string;
     parameters: { [index: string]: any };
+    minimumEqualToMaximum: boolean;
```

Comment:
I added this `isMinimumEqualToMaximum()` function, I don't know why it created a variable, is there anyway to avoid this?

Reply:
fixed it with `@JsonIgnore`